api/lib/resolvers.js

```diff
@@ -401,11 +408,13 @@ const resolvers = (networkList) => ({
       { dataSources }
     ) => {
       await localStore(dataSources, networkId).dataReady
+      const network = await database(config)('').getNetwork(networkId)
```

Comment:
networks are already loaded, let's not request them from the db again

Reply:
XD

Reply:
send them from `networkList`you mean? Would mean a significant refactor in resolvers

Reply:
Or maybe not that significant, but yes, wondering if there is an easier way

Reply:
Yep, get them from networklist

Reply:
oh, actually it is very easy because it is insider resolvers, now I see